src/PlexRichPresence.DiscordRichPresence/Rendering/SerieSessionRenderer.cs

```diff
@@ -12,8 +12,8 @@ public override RichPresence RenderSession(PlexSession session)
         return new RichPresence
         {
             Type = ActivityType.Watching,
-            StatusDisplay = StatusDisplayType.Details,
-            Details = $"{session.MediaTitle}",
+            StatusDisplay = StatusDisplayType.State,
```

Comment:
Would you agree to add a test to reflect these changes in `PlexSessionRenderingServiceTests` ?
Otherwise I can do it :+1:

Reply:
Yup, will do, tho dunno when I'll get to it